packages/babel-plugin-transform-typescript/test/fixtures/enum/outer-const-foldable/output.js

```diff
@@ -0,0 +1,15 @@
+// ts 5.0 feature
+
+const BaseValue = 10;
+const Prefix = "/data";
+var Values; // 12
+(function (Values) {
+  Values[Values["First"] = 10] = "First";
+  Values[Values["Second"] = 11] = "Second";
+  Values[Values["Third"] = 12] = "Third";
+})(Values || (Values = {}));
```

Comment:
I also want to optimize the enum layout.
Since the snapshot changes a lot, I will put it in another PR.